### PR TITLE
deploy core instance using operator

### PR DESCRIPTION
**Pull request by nicolasparada**
Changed the CLI command to use the operator.
Also updated the docker base image (node.js) and the CLI.

Noticed that Vivo is not working. We need to fix it on a next PR.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205171166326738

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
+  "proxy": "https://cloud-api-dev.calyptia.com",
```
Comment (patrick-stephens) [inline ui/package.json:25]: should this be prod?

File: ui/src/components/DeployCoreInstance.tsx [hunk 2]
```diff
@@ -25,13 +25,35 @@ export default function DeployCoreInstance() {
         const args = [
             "apply",
             "--context", "docker-desktop",
-            "-f", "https://raw.githubusercontent.com/calyptia/vivo/master/vivo.yaml"
+            "-f", "https://raw.githubusercontent.com/calyptia/vivo/master/vivo-deployment.yaml"
```
Comment (patrick-stephens) [inline ui/src/components/DeployCoreInstance.tsx:28]: Vivo has changed quite a bit to support K8S so hopefully should be simpler now.
I have a deployment in on-prem for it: https://github.com/calyptia/chart-cloud-standalone/blob/main/chart/templates/vivo/deployment-vivo.yaml

File: ui/package.json (hunk 1 above)
Comment (nicolasparada) [reply ui/package.json:25]: This proxy is just for the development server of CRA (create react app).

File: Dockerfile [hunk 3]
```diff
@@ -47,20 +47,20 @@ RUN apk add --no-cache curl
...
-    && curl -sSfL https://github.com/calyptia/cli/releases/download/v0.41.1/cli_0.41.1_linux_amd64.tar.gz | tar -xz \
+    && curl -sSfL https://github.com/calyptia/cli/releases/download/v1.4.9/cli_1.4.9_linux_amd64.tar.gz | tar -xz \
     && chmod +x ./calyptia && mv ./calyptia /usr/local/bin/calyptia \
     && mkdir -p /linux \
     && cp /usr/local/bin/kubectl /linux/ \
     && cp /usr/local/bin/calyptia /linux/
 
 RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl" \
-    && curl -sSfL https://github.com/calyptia/cli/releases/download/v0.41.1/cli_0.41.1_darwin_amd64.tar.gz | tar -xz \
+    && curl -sSfL https://github.com/calyptia/cli/releases/download/v1.4.9/cli_1.4.9_darwin_amd64.tar.gz | tar -xz \
```
Comment (patrick-stephens) [inline Dockerfile:57]: Probably bump these up to 1.6.1 now to handle the internal/external cluster URL and other fixes

Comment (nicolasparada) [reply Dockerfile]: Done

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**